IConsoleReader.cs

```diff
@@ -0,0 +1,7 @@
+namespace TicTacToe
+{
+    public interface IConsoleReader
```

Comment:
More a comment than something you have to absolutely change right now: while it's often a convention to name interfaces `IClassName` or `ClassNameImpl`, those names aren't very useful or descriptive. It's useful to think about what role the interface is playing: e.g. if you had a second class in addition to `ConsoleReader` implementing `IConsoleReader`, what would it look like? Perhaps this interface functions more generally as something that delivers user input?